src/app/public/modules/colorpicker/types/colorpicker-axis.ts

```diff
@@ -1,8 +1,26 @@
 // spell-checker:ignore Colorpicker,
+/**
+ * Specifies the
+ */
 export interface SkyColorpickerChangeAxis {
```

Comment:
I'm not sure why this property is part of the public API... its only used in the `SkyColorpickerSliderDirective` (which is internal), and as a setter in the colorpicker component itself - but not an`@Input()` setter. @Blackbaud-SteveBrush could you shed some light on this? Do we need to document?

Reply:
Not sure if Steve is around today. It sounds like it shouldn't be in the docs, so I'm going to go ahead and add `@internal`. But I'll leave this comment thread open to make sure we confirm with Steve before merging.

Reply:
Yep, internalize it.